test/lib/rails-jquery-autocomplete_test.rb

```diff
@@ -3,7 +3,7 @@
 module RailsJQueryAutocomplete
   class RailsJQueryAutocompleteTest < ActionController::TestCase
     ActorsController = Class.new(ActionController::Base)
-    ActorsController.autocomplete(:movie, :name)
+    ActorsController.autocomplete(:action_name, { :movie => :name }, { :display_value => :name })
```

Comment:
Line is too long. [97/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.